### ISSUE TITLE
Ensure max. standards of accessibility of all code

### ISSUE THREAD

**Issue by eghale** [TechforgoodCAST/cast-website#92]
This should be a key design consideration. 

Tips here https://abilitynet.org.uk/news-blogs/6-quick-checks-website-accessibility?dm_i=3SCI,5QT6,2LWS0N,JKTT,1

And here http://a11yproject.com/

**Comment by eghale** [comment]
Also here https://www.w3.org/WAI/WCAG20/quickref/

**Comment by eghale** [comment]
And here https://www.youtube.com/watch?v=8Ik_LHmZx8Y&app=desktop
